README.md

```diff
@@ -22,8 +25,52 @@ The script serves as installer by default, but can also be used with flags `--up
 
 The agent can run as a standalone binary, as long as configuration files are set up correctly.
 
+### Systemd (non-root)
+These instructions require running Caddyserver as Docker proxy. Before proceeding, make sure a proxy is listening on a `DOCKER_HOST` address and forwards requests to the host Docker daemon non-networked socket. You can find instructions about a recommended setup [here](#using-a-docker-reverse-proxy).
+
+To install and run the agent as a systemd service without adding `metrikad` user to `docker` group, use the following one-liner:
+```bash
+DOCKER_HOST=tcp://127.0.0.1:2379 DOCKER_API_VERSION=1.41 MA_BLOCKCHAIN={blockchain} MA_API_KEY={api_key} bash -c "$(curl -sL https://raw.githubusercontent.com/Metrika-Inc/agent/main/install.sh) --no-docker-grp"
```

Comment:
Have you tested this? I tried a similar thing with `--upgrade` flag and it did not work IIRC.

Reply:
Could you paste the one-liner you used and the resulting systemd unit?

Reply:
Addressed in 15c4b7c.